tests/check/check_h26xsigned_auth.c

```diff
@@ -58,6 +58,9 @@ struct validation_stats {
   bool public_key_has_changed;
 };
 
+static signed_video_t *
+generate_and_set_private_key_on_camera_side(struct sv_setting settings, bool add_public_key_to_sei, nalu_list_item_t *i_nalu, nalu_list_item_t **sei);
```

Comment:
No need to declare here. We haven't done that for other helpers. Try to keep them close to the test instead.